validator/sawtooth_validator/journal/chain.py

```diff
@@ -305,6 +323,7 @@ def __init__(self,
                  consensus_module,
                  block_cache,
                  block_sender,
+                 state_view_factory,
```

Comment:
Use the addition of parameters as a opportunity to update the params in the doc string.